phpstan.neon

```diff
@@ -16,3 +16,6 @@ parameters:
 
         # Test dependencies
         - 'tests/Application/**/*'
+
+        # Menu Provider
+        - 'src/Menu/PageUrlProvider.php'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** instead of ignoring the errors in the class, you could ignore the error‏

Reply:
Can we see that later with a recent version of PHP Stan ?

Reply:
of course! as seen together, it's an error that can't be ignored...